client/e2e/workflows/bceidbusiness/industry_user.spec.ts

```diff
@@ -0,0 +1,222 @@
+// 🧪 Suite to test the bceidbusiness new user workflow using storageState
+// 🔍 Asserts new user is redirected to profile
+
+import { test, expect, APIResponse } from "@playwright/test";
+// 🪄 Page Object Models
+import { DashboardPOM } from "@/e2e/poms/dashboard";
+// ℹ️ Environment variables
+import * as dotenv from "dotenv";
+import { OperatorPOM } from "@/e2e/poms/operator";
+import { fieldsUpdate } from "@/e2e/utils/helpers";
+import { deleteUserOperatorRecord } from "@/e2e/utils/queries";
+dotenv.config({ path: "./e2e/.env.local" });
+const happoPlaywright = require("happo-playwright");
+
+test.beforeEach(async ({ context }) => {
+  await happoPlaywright.init(context);
+  let response: APIResponse = await context.request.get(
+    "http://localhost:8000/api/registration/test-setup",
+  );
+  // Wait for the response and check for success status text and code (e.g., 200)
+  expect(await response.text()).toBe("Test setup complete.");
+  expect(response.status()).toBe(200);
+
+  await deleteUserOperatorRecord(process.env.E2E_INDUSTRY_USER_GUID as string);
```

Comment:
Once @Sepehr-Sobhani's PR editing the `test-endpoint` goes in, there might be a more elegant way to do this. Currently this loads all the fixtures and then deletes the current user_operator, but it would save time to just not load the current user in the first place